### ISSUE TITLE
Chia-gigahorse build for windows

### ISSUE THREAD

**Issue by akesman** [madMAx43v3r/chia-gigahorse#2]
Hello Max,

Can you tell me where can I download a ready-made build for windows?

Thank you

**Comment by madMAx43v3r** [comment]
It's not available yet, but you can use WSL.

**Comment by Sergiosen** [comment]
Hi, Max!  I tried to install the Gigahorse node on win 10 LTSC, I get an "import error" with any command in the terminal.  Is my Windows missing some package?
![Screenshot_20230218_201553](https://user-images.githubusercontent.com/125818266/219961387-96b85da2-f01b-4080-9bac-ebe4753cb500.jpg)

**Comment by madMAx43v3r** [comment]
Is this the latest version? 
chia-gigahorse-farmer-1.6.2.giga4-windows-2.zip

**Comment by Sergiosen** [comment]
yes

**Comment by Sergiosen** [comment]
in my Windows assembly from lex_6000, some necessary packages may be missing, probably?  Or some paths need to be registered.

**Comment by madMAx43v3r** [comment]
Yes I think you're missing Microsoft Visual C++ Redistributable: https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170

**Comment by Sergiosen** [comment]
Max, thank you very much, installed, everything worked!  

The only proof of space check of k32 plots with C9 compression is unsuccessful:  many bad allocation, especially when the MMX harvester is running, and plots with C7 compression are checked normally.  Moreover, the processes consume few resources of the video card. I am using 3060 graphics card 12Gb, e5 xeon 2678 v3 processor, RAM 96Gb.

**Comment by madMAx43v3r** [comment]
C9 needs a lot of RAM / VRAM
